frontend/src/routes/Home/Search/Details/DetailsOverviewPage.tsx

```diff
@@ -350,7 +350,12 @@ export default function DetailsOverviewPage(props: {
         <PageSection variant={'light'}>
           <Stack hasGutter>
             <Text style={{ fontSize: '1.25rem', fontFamily: 'RedHatDisplay' }} component={'h2'}>
-              {`${resource.kind} details`}
+              <Trans
+                i18nKey="search.resource.details"
+                values={{
+                  resource: resource.kind,
+                }}
+              />
```

Comment:
No need to use `<Trans>` component if the string does not contain styling. You could just do:
```
{t('search.resouce.details', { resource: resource.kind })}
```